app/src/androidTest/java/com/github/ybecker/epforuml/database/FirebaseDatabaseAdapterTest.kt

```diff
@@ -456,4 +456,26 @@ class FirebaseDatabaseAdapterTest {
             assertThat(it, equalTo(listOf(romain.userId)))
         }.join()
     }
+
```

Comment:
iÎt would be good to test other scenarios, such as what happens if you try to add the same connection twice, or if you try to remove a connection that doesn't exist.